contents/docs/user-guides/organizations-and-projects.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Organizations and projects
+sidebar: Docs
+showTitle: true
+---
+
+PostHog gives you tools for data access control and logical separation of data: organizations and projects.
+
+## Organizations
+
+An organization is the largest building block of PostHog's structure: it's made up of projects and has PostHog users as members. 
+Each organization member has an organization-wide [access level](#access-levels) dictating their ability to perform certain actions.
+
```

Comment:
Maybe worth asking @lottiecoxon for a simple art here that represents the hierarchy of PostHog instance -> org -> project (and members)?